Mathlib/Analysis/Normed/Lp/PiLp.lean

```diff
@@ -675,6 +675,19 @@ instance instIsBoundedSMul [SeminormedRing 𝕜] [∀ i, SeminormedAddCommGroup
       simp_rw [← NNReal.mul_rpow, smul_apply]
       exact Finset.sum_le_sum fun i _ => NNReal.rpow_le_rpow (nnnorm_smul_le _ _) hp0.le
 
+instance instNormSMulClass [SeminormedRing 𝕜] [∀ i, SeminormedAddCommGroup (β i)]
```

Comment:
This worsens an existing code duplication: the existing `PiLp.instIsBoundedSMul` and `WithLp.instProdIsBoundedSMul` are already very similar to each other, and the `NormSMul` instances added in this PR use almost the same proof again for `NormSMulClass`. But I don't see a natural way of avoiding it unfortunately.

Reply:
Yes, this annoyed me too

Reply:
Since neither of us has a better solution we’ll have to tolerate it fir now